test/evergreen.yml

```diff
@@ -1187,6 +1187,30 @@ tasks:
           test_config_filename: configs/bounded_cursor_perf_default.txt
           test_name: bounded_cursor_perf
 
+  - name: cppsuite-cursor-bound-01-default
+    tags: ["pull_request"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "cppsuite test"
+        vars:
+          test_config: debug_mode=(cursor_copy=true)
```

Comment:
Question: what is this option used for?

Reply:
Prevents invalid uses of wiredtiger buffers which is what happened with the cursor search range_srch_key char *.

Reply:
Should we add this to the stress config as well?

Reply:
No its not very performant.